packages/core/src/precomputation.ts

```diff
@@ -32,8 +32,7 @@ export class PrecomputationMMR extends CoreMMR {
     const parentMmr = new CoreMMR(store, hasher, parentMmrUuid);
     const elementsCount = await parentMmr.elementsCount.get();
     const leavesCount = await parentMmr.leavesCount.get();
-    const rootHash = await parentMmr.rootHash.get();
-
+    const rootHash = (await parentMmr.rootHash.get()) ?? "0x0";
```

Comment:
```suggestion
    const rootHash = await parentMmr.rootHash.get();
```

and then below do:
```ts
if(rootHash) await precomputationMMR.rootHash.set(rootHash);
```
or 
```ts
if(rootHash !== <value when .get() doesnt find anything>) await precomputationMMR.rootHash.set(rootHash);
```

Reply:
This way the behaviour of the precomputation MMR is kept the same as the behaviour of the normal MMR